### PR TITLE
#612 `upload-area` component not clearing data

### PR DESCRIPTION
**Pull request by beneditomauro**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-components-vue/issues/612 |
| Decisions | -Move common logic to the `upload` mixin |
| Animated GIF | T0yp5QhkP8 |

### CODE REVIEW

**Review by 3rdvision — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
-*
+* `upload-area` component not clearing data - [#612](https://github.com/ripe-tech/ripe-util-vue/issues/612)
```
Comment (joao-conde) [inline CHANGELOG.md:20]: ripe-util-vue/#612

**Review by joao-conde — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (beneditomauro) [reply CHANGELOG.md:20]: > ripe-util-vue/#612

This is a component related bug,

Comment (joao-conde) [reply CHANGELOG.md:20]: Link is wrong then

**Review by joamag — APPROVED** [review]

**Comment by joamag** [comment]
@beneditomauro good fix 👍

**Pull request merged**